Crashlytics/Crashlytics/Components/FIRCLSUserLogging.h

```diff
@@ -31,6 +31,8 @@ extern NSString* const FIRCLSUserNameKey;
 extern NSString* const FIRCLSUserEmailKey;
 extern NSString* const FIRCLSDevelopmentPlatformNameKey;
 extern NSString* const FIRCLSDevelopmentPlatformVersionKey;
+extern NSString* const FIRCLSOnDemandRecordedExceptionsKey;
+extern NSString* const FIRCLSOnDemandDroppedExceptionsKey;
```

Comment:
Are we committed to using "Exceptions" as the name. Should we use "Events"?

Reply:
Since there's already a cl submitted with this to change the Procella schema, let's leave it as exception